api/clinics.go

```diff
@@ -250,6 +251,20 @@ func (h *Handler) UpdateTier(ec echo.Context, clinicId ClinicId) error {
 	return ec.NoContent(http.StatusOK)
 }
 
+func (h *Handler) UpdateSuppressedNotifications(ec echo.Context, clinicId ClinicId) error {
+	ctx := ec.Request().Context()
+	dto := UpdateSuppressedNotifications{}
```

Comment:
Stylistic nitpick, https://google.github.io/styleguide/go/best-practices#non-pointer-zero-values
Prefer var declarations for variables that are using the zero value.
`var dto UpdateSuppressedNotifications`